src/App.js

```diff
@@ -1,13 +1,15 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { BrowserRouter as Router, Route } from 'react-router-dom';
-import About from './components/support';
+import About from './components/Support';
 import Home from './components/Home';
-import './components/home.css';
+import './components/Home.css';
+import './components/productsAvailable/components/styles.css';
+import './components/rows/components/style.css';
```

Comment:
you are already importing the `styles.css` in the appropriate `index.js`. You don't need to import them again here. delete `line 6-7`